devito/ir/iet/nodes.py

```diff
@@ -156,6 +157,24 @@ def __init__(self, name, params=None):
     def __repr__(self):
         return "Call::\n\t%s(...)" % self.name
 
+    @property
+    def functions(self):
+        """Return all :class:`Symbol` objects used by this :class:`Call`."""
+        return tuple(p for p in self.params if isinstance(p, types.AbstractFunction))
```

Comment:
``... if p.is_AbstractFunction`` ?

Reply:
Nope, the DLE insert actual expressions into the call signature, `self.params` is not guaranteed to be internal symbols only.

Reply:
OK